app/page.tsx

```diff
@@ -176,18 +178,18 @@ export default function Home() {
                   <div key={index} className="flex mt-4 items-center" >
                     <div>
                       <p className={`${commonTextClass}`}>
-                        {formatSessionTime(data.start_time)}
+                        {formatQuizSessionTime(data.start_time)}
                       </p>
                       <p className={`${commonTextClass}`}>
-                        {formatSessionTime(data.end_time)}
+                        {formatQuizSessionTime(data.end_time)}
                       </p>
                     </div>
                     <div className="bg-card rounded-lg shadow-lg min-h-24 h-auto py-6 relative w-full flex flex-row justify-between mr-4">
                       <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0  rounded-s-md`}></div>
                       <div className="text-sm md:text-base font-semibold mx-6 md:mx-8">
-                        <span className="font-normal pr-4">Subject:</span> {data.meta_data.stream}
+                        <span className="font-normal pr-4">Subject:</span> {data.name}
```

Comment:
"Subject" should be data.subject

Reply:
I can't find `subject` as a key in response. The reason why I used `data.name` is because every quiz name also consists subject name.